server/controllers/tasksController.test.ts

```diff
@@ -59,7 +59,7 @@ describe('TasksController', () => {
     beforeEach(() => {
       when(cruManagementAreaService.getCruManagementAreas).calledWith(token).mockResolvedValue(cruManagementAreas)
       when(taskService.getAll).calledWith(expect.anything()).mockResolvedValue(paginatedResponse)
-      when(userService.getUserList).calledWith(token, ['assessor', 'matcher']).mockResolvedValue(users)
+      when(userService.getUserList).calledWith(token, ['assessor', 'appeals_manager']).mockResolvedValue(users)
```

Comment:
Is the `appeals_manager` role a direct replacement for `matcher`?  They sound quite different!

Reply:
No, it isn't -- these changes are more to do with unreferencing the `matcher` role: the actual roles used in these instances don't matter (these tests only check presence/absence of any given role, and have nothing to do with actual permissions), so `matcher` was replaced with another role that will most likely continue to exist!

Reply:
Makes sense.  Thanks for the explanation.